modules/survey_accounts/jsx/columnFormatter.js

```diff
@@ -0,0 +1,14 @@
+function formatColumn(column, cell, rowData, rowHeaders) {
+
+
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+  }, this);
+    if(column === 'URL') {
+        var url = loris.BaseURL + "/survey.php?key=" + row['URL'];
```

Comment:
@kongtiaowang Please run ESLint on the Javascript files before committing to have a standard formatting.